css-four-column-layout/styles.css

```diff
@@ -0,0 +1,54 @@
+body {
+  background-color: #f7f8fa;
+  font-family: 'Barlow', sans-serif;
+  text-align: center;
+}
+
+.container {
+  width: 1000px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.row {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
+
+h1 {
+  width: 100%;
+  font-size: 1.3rem;
+  font-weight: 800;
+}
+
+h2 {
+  font-size: 0.75rem;
+  font-weight: 800;
+}
+
+p {
+  font-size: 0.75rem;
+  font-weight: 500;
+}
+
+.column-fourth {
+  width: 22%;
+  display: flex;
+  flex-direction: column;
+}
+
+.img-container-icon {
+  width: 25%;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.img-container-screen > img {
+  max-width: 100%;
+}
+
+.img-container-icon > img {
+  max-width: 100%;
+}
```

Comment:
🤔 I wonder if you can't reorganize this so that you can just apply the classes directly to the images? Child selectors are kinda brittle.

Reply:
Thank you for the input. I'll clean this up! I would love a little more explanation on what you mean by brittle?